offload/unittests/OffloadAPI/event/olGetEventInfoSize.cpp

```diff
@@ -19,6 +19,12 @@ TEST_P(olGetEventInfoSizeTest, SuccessQueue) {
   ASSERT_EQ(Size, sizeof(ol_queue_handle_t));
 }
 
+TEST_P(olGetEventInfoSizeTest, SuccessIsComplete) {
+  size_t Size = 0;
+  ASSERT_SUCCESS(olGetEventInfoSize(Event, OL_EVENT_INFO_IS_COMPLETE, &Size));
+  ASSERT_EQ(Size, sizeof(bool));
```

Comment:
```suggestion
  EXPECT_EQ(Size, sizeof(bool));
```

Reply:
Any particular reason why? We don't use `ASSERT` anywhere else in the unit tests and it doesn't make sense for the function to continue running if this fails.

Reply:
In which case, if we want to change over to that, I think it makes sense to do that as a single separate change that touches all the tests. I think we should use ASSERT here now for the sake of consistency with other tests.